docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -0,0 +1,228 @@
+USE_SECURE_CERT=FALSE
+```
+
+## 4 Set up local Provisioning Certificate Caching Service (PCCS)
+### 4.1 Register an Intel developer account
+
+and subscribe to get the subscription key. This value will be used during PCCS service bring up.
+
+If you don't have an Intel account. Go to https://www.intel.com to register one by using the button at the corner.
+
+![Intel Account](./images/intel-account.png)
+
+Sign in with your Intel account and then go to
+https://api.portal.trustedservices.intel.com/provisioning-certification
+
+You shall see the screen like this:
+
+![Subscribe1](./images/subscribe1.png)
+
+Click the subscribe button then you shall see page like this.
+
+![Subscribe2](./images/subscribe2.png)
+
+Then click subscribe button again. You shall see your subscription information.
+
+![Primary Key](./images/primary-key.png)
+
+Get the value of your primary key, which will be used later.
+
+### 4.2 Install PCCS by using its Debian package
+Install nodejs and npm if you haven't
+
+```bash
+curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash
+sudo apt-get install -y nodejs
+```
+
+This [link](https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/web/sgx-dcap-pccs/) lists all different version of pccs, please download the most recent version that matches your OS version.
+
+In this document, we use sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb as an example, run the command blow to download the package
+```bash
+cd ~
+wget https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/web/sgx-dcap-pccs/sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb
+```
+
+Then install the package. You will be asked to finish the configuration during the installation process.
+```bash
+sudo dpkg -i sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb
+```
+![Config](./images/config.png)
+
+Recommended config:
+- HTTPS listening port: use default value.
+- Set the PCCS service to accept local connections only: use default value.
+- Set your Intel PCS API key: use the primary key of your subscription.
+- Choose caching fill method: use default value.
+- Set PCCS server administrator password: set your password
+- Set PCCS server user password: set your password
+- Do you want to generate insecure HTTPS key and cert for PCCS service?: Use default value and then in the following questions type in your info.
+
+You can skip the following two items.
+- A challenge password []:
+- An optional company name []:
+
+Then check the status of your service.
+```bash
+pm2 status
+```
+You should be able to see the service is running.
+![nodejs](./images/nodejs.png)
+
+Run the following command to verify if it can actually fetch the root ca crl from Intel PCK service
+```bash
+curl --noproxy "*" -v -k -G "https://localhost:8081/sgx/certification/v2/rootcacrl"
+```
+
+To learn more about PCCS, please refer to this [link](https://github.com/intel/SGXDataCenterAttestationPrimitives/tree/master/QuoteGeneration/pccs).
```

Comment:
```suggestion
To learn more about PCCS, please refer to the [PCCS GitHub repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/tree/master/QuoteGeneration/pccs).
```

Explanation: "link", "here", etc should be avoided as link names for accessibility reasons, among others, as discussed in the [W3C guidance](https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text).